src/target/cortexm.c

```diff
@@ -1062,6 +1062,25 @@ static uint32_t cortexm_dwt_func(target_s *target, target_breakwatch_e type)
 	}
 }
 
+static uint32_t cortexm_dwt_func_v8m(target_s *target, target_breakwatch_e type, size_t len)
```

Comment:
Similar comment as in the header about `_v8m` vs `dwtv2` - please prefer the latter.